Order.API.Test/Controllers.Test/ControllerTest.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace Order.API.Test.Controllers.Test
+{
+    using Moq;
+    using Order.API.Controllers;
+    using Order.Domain.Common;
+    using Order.Domain.Common.Resources;
+    using Order.Domain.DTOs;
+    using Order.Domain.Entities;
+    using Order.Domain.Services;
+    using System.Resources;
+
+    public class ControllerTest
+    {
+        private readonly Mock<IProductService> _productService;
+        public readonly ProductDTO _request;
+        public readonly ProductController _productController;
+        public readonly string productSuccessMsg;
+        public readonly string productFailureMsg;
+        public ControllerTest() 
+        {
+            _productService = new Mock<IProductService>();
+            _productController = new ProductController(_productService.Object);
+            _request = new ProductDTO() { Name = "TV", Description = "OLED TV", Price = 350000 };
+            ResourceManager resourceManager = new ResourceManager(ResourceKeys.SystemMsgResrceName, typeof(MessageResources).Assembly);
+            productSuccessMsg = resourceManager?.GetString(ResourceKeys.ProductSuccess) ?? string.Empty;           
+            productFailureMsg = resourceManager?.GetString(ResourceKeys.ProductFailure) ?? string.Empty;
```

Comment:
why make it so complicated, make the regex pub generated and directly use the value